tinygrad/renderer/cstyle.py

```diff
@@ -104,10 +104,10 @@ def ssa(prefix:str, u:Optional[UOp]=None):
       c[prefix] += 1
       return ret
 
-    child_count = Counter(v for ru in uops for v in ru.vin)
+    child_count = Counter(v for ru in uops for v in ru.src)
 
     for u in uops:
-      uop,dtype,vin,args = u.uop,u.dtype,u.vin,u.arg
+      uop,dtype,vin,args = u.op,u.dtype,u.src,u.arg
```

Comment:
why is cstyle sitll using uop and vin

Reply:
I think renaming that is a bigger change than it needs to be. The smaller you can make PRs the better.